### PR TITLE
tsc: jsdoc import hammerjs for controllers event.

### PR DESCRIPTION
**Pull request by peaceshi**

### CODE REVIEW

File: modules/core/package.json [hunk 1]
```diff
@@ -39,5 +39,8 @@
     "math.gl": "^3.4.2",
     "mjolnir.js": "^2.5.0",
     "probe.gl": "^3.3.1"
+  },
+  "devDependencies": {
```
Comment (ibgreen) [inline modules/core/package.json:43]: Longer term, I wonder if this coild be imported and re-exported from the mjolnir.js module instead, as that is where the hammerjs dependency is.

**Review by ibgreen — APPROVED** [review]

File: modules/core/package.json (hunk 1 above)
Comment (peaceshi) [reply modules/core/package.json:43]: seems like ok. but mjolnir.js do not have type definition file ....
import form hammerjs , has types file ✓
import form mjolnir.js, no types file ×
or just add a standalone d.ts (not recommend. if code changed, need to change it every time)

Comment (Pessimistress) [inline modules/core/package.json:43]: What does this do for the users? `devDependencies` are not installed when you install this package from npm.

File: modules/core/src/controllers/controller.js [hunk 2]
```diff
@@ -21,6 +21,11 @@
 /* eslint-disable max-statements, complexity */
 import TransitionManager from './transition-manager';
 
+/**
+ *
+ * @typedef { import('hammerjs')}
```
Comment (Pessimistress) [inline modules/core/src/controllers/controller.js:26]: We are not using JSDoc to define types. All type definitions should go into .d.ts files.

**Pull request closed without merging**